lm_eval/api/task.py

```diff
@@ -437,14 +436,15 @@ def build_all_requests(
                 system_instruction,
                 apply_chat_template,
                 fewshot_as_multiturn,
-                chat_template,
+                lm,
             )
 
             # TODO: we should override self.config.repeats if doing greedy gen so users don't waste time+compute
             inst = self.construct_requests(
                 doc=doc,
                 ctx=fewshot_ctx,
                 metadata=(self.config["task"], doc_id, self.config.repeats),
+                apply_chat_template=apply_chat_template
```

Comment:
this will be used to remove prepended whitespace on continuations in multichoice tasks